project/pick_estate.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="initial-scale=1.0, user-scalable=no">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/maps.css">
+    <title>Pick your estate</title>
+</head>
+
+<body>
+    <div id="map"></div>
+    <div id="capture"></div>
+    <script src='pick_estate.js'></script>
+    <script async defer
+        src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDXWlNQI0L23IiD5_QT6T4gkCDKz4OI5w8&callback=initMap">
+        </script>
+</body>
+
+</html>
```

Comment:
I'm glad you are keeping this a separate page. I'm changing index.html big time for styling

Reply:
Thought you might do that. Maps was a fairly large change too, so seemed better to be a separate page.